core/templates/services/state-interaction-stats.service.ts

```diff
@@ -34,34 +32,26 @@ import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service';
 
 export interface IAnswerData {
-  /* eslint-disable camelcase */
-  answer; // Type depends on interaction id.
-  frequency: number;
-  is_addressed?: boolean;
-  /* eslint-enable camelcase */
+  'answer': any; // Needs to be 'any' because type depends on interaction id.
```

Comment:
@nishantwrp @vojtechjelinek  Is it possible to define a custom type for this, that admits a form that's something like "boolean | string | number | ..." (such that it's a whitelist rather than "any")?

Also, I assume it's fine for @brianrodri to check in this PR now, and you'll take care of this in a future pass?

Reply:
Yeah, i think we can define it's type using `|` because there are a fixed number of different cases possible. I think it's type would be similar to
https://github.com/oppia/oppia/blob/66597a96525e24fe4f01f2ffe9c1a5d929c67dca/core/templates/domain/exploration/RuleObjectFactory.ts#L37-L39
i'm not sure though.

Reply:
Done.